apps/prairielearn/src/pages/publicInstructorAssessmentQuestions/publicInstructorAssessmentQuestions.html.ts

```diff
@@ -0,0 +1,221 @@
+import { z } from 'zod';
+
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { HeadContents } from '../../components/HeadContents.html.js';
+import { AssessmentBadge } from '../../components/AssessmentBadge.html.js';
+import { TagBadgeList } from '../../components/TagBadge.html.js';
+import { TopicBadge } from '../../components/TopicBadge.html.js';
+import { compiledScriptTag } from '../../lib/assets.js';
+import {
+  AlternativeGroupSchema,
+  AssessmentQuestionSchema,
+  AssessmentsFormatForQuestionSchema,
+  QuestionSchema,
+  TagSchema,
+  TopicSchema,
+  ZoneSchema,
+} from '../../lib/db-types.js';
+
+export const AssessmentQuestionRowSchema = AssessmentQuestionSchema.extend({
+  alternative_group_number_choose: AlternativeGroupSchema.shape.number_choose,
+  alternative_group_number: AlternativeGroupSchema.shape.number,
+  alternative_group_size: z.number(),
+  assessment_question_advance_score_perc: AlternativeGroupSchema.shape.advance_score_perc,
+  display_name: z.string().nullable(),
+  number: z.string().nullable(),
+  open_issue_count: z.string().nullable(),
+  other_assessments: AssessmentsFormatForQuestionSchema.nullable(),
+  sync_errors_ansified: z.string().optional(),
+  sync_errors: QuestionSchema.shape.sync_errors,
+  sync_warnings_ansified: z.string().optional(),
+  sync_warnings: QuestionSchema.shape.sync_warnings,
+  topic: TopicSchema,
+  qid: QuestionSchema.shape.qid,
+  start_new_zone: z.boolean().nullable(),
+  start_new_alternative_group: z.boolean().nullable(),
+  tags: TagSchema.pick({ color: true, id: true, name: true }).array().nullable(),
+  title: QuestionSchema.shape.title,
+  zone_best_questions: ZoneSchema.shape.best_questions,
+  zone_has_best_questions: z.boolean().nullable(),
+  zone_has_max_points: z.boolean().nullable(),
+  zone_max_points: ZoneSchema.shape.max_points,
+  zone_number_choose: ZoneSchema.shape.number_choose,
+  zone_number: ZoneSchema.shape.number,
+  zone_title: ZoneSchema.shape.title,
+});
+type AssessmentQuestionRow = z.infer<typeof AssessmentQuestionRowSchema>;
+
+export function InstructorAssessmentQuestions({
+  resLocals,
+  questions,
+}: {
+  resLocals: Record<string, any>;
+  questions: AssessmentQuestionRow[];
+}) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${HeadContents({ resLocals })}
+        ${compiledScriptTag('publicInstructorAssessmentQuestionsClient.ts')}
+      </head>
+      <body>
+        ${renderEjs(import.meta.url, "<%- include('../partials/navbar'); %>", resLocals)}
+        <main id="content" class="container-fluid">
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white d-flex align-items-center">
+              ${resLocals.assessment.title} ${resLocals.assessment.number}: Questions
+            </div>
```

Comment:
the whole new way of doing the Navbar was just merged, so you'll have to update this here. 

You will need to merge master and then check out this PR to see how it works on other pages: https://github.com/PrairieLearn/PrairieLearn/pull/10277